flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/FlinkConfigManager.java

```diff
@@ -177,6 +180,17 @@ private void addOperatorConfigsFromSpec(AbstractFlinkSpec spec, Configuration co
         }
     }
 
+    @SuppressWarnings("unchecked")
+    private void applyConfigsFromCurrentSpec(
```

Comment:
This method should internally call the `addOperatorConfigsFromSpec` so we keep things tidy and logically together.

Reply:
Moved.